src/components/Filters/Filters.stories.tsx

```diff
@@ -12,139 +14,210 @@ export default {
     badges: [BADGE.BETA],
   },
   args: {
-    checkboxFields: [
-      {
-        legend: 'Filters Segment 1',
-        checkboxes: [
-          { label: 'Filters label 1', value: 'Filters label 1' },
-          { label: 'Filters label 2', value: 'Filters label 2' },
-          { label: 'Filters label 3', value: 'Filters label 3' },
-        ],
-      },
-    ],
-    onClose: (checkedValues) => {
-      console.log(checkedValues);
-    },
+    buttonText: 'Filters',
+    buttonStatusVariant: 'neutral',
+    children: (
+      <Filters.FiltersCheckboxField legend="Filters Segment 1">
+        <Checkbox
+          label="Filters label 1"
+          onChange={
+            () => {} /* eslint-disable-line @typescript-eslint/no-empty-function */
+          }
+        />
+        <Checkbox
+          label="Filters label 2"
+          onChange={
+            () => {} /* eslint-disable-line @typescript-eslint/no-empty-function */
+          }
+        />
+        <Checkbox
+          label="Filters label 3"
+          onChange={
+            () => {} /* eslint-disable-line @typescript-eslint/no-empty-function */
+          }
+        />
+      </Filters.FiltersCheckboxField>
+    ),
   },
+  decorators: [
+    (Story) => (
+      <div style={{ margin: '0.25rem', height: '100vh' }}>
```

Comment:
`100vh` is required for chromatic to create the snaps properly, or just cuts off at the Button height for the drawer